src/Security/Profile.php

```diff
@@ -8,151 +8,43 @@
  * Class Profile
  * @package kuzzleio/kuzzle-sdk
  */
-class Profile extends Document
+class Profile
 {
-    protected $deleteActionName = 'deleteProfile';
-
-    protected $updateActionName = 'updateProfile';
-
-    protected $saveActionName = 'createOrReplaceProfile';
-
     /**
-     * @var Policy[]
-     */
-    protected $policies;
-
-    /**
-     * Role constructor.
-     * @param Security $kuzzleSecurity An instantiated Kuzzle\Security object
-     * @param string $id Unique profile identifier
-     * @param array $content Profile content
-     * @param array $meta Profile metadata
+     * @var Security The kuzzle security instance associated to this document
      */
-    public function __construct(Security $kuzzleSecurity, $id = '', array $content = [], array $meta = [])
-    {
-        parent::__construct($kuzzleSecurity, $id, $content, $meta);
-
-        $this->syncPolicies();
-
-        return $this;
-    }
+    public $security;
 
     /**
-     * Adds a policy to the profile.
-     *
-     * @param array|Policy $policy Unique Kuzzle\Security\Policy instance corresponding to the new associated policy
-     * @return Profile
+     * @var string Unique document identifier
      */
-    public function addPolicy($policy)
-    {
-        $this->policies[] = $this->policyFactory($policy);
-
-        return $this;
-    }
+    public $id;
 
     /**
-     * Returns this profile associated roles.
-     *
-     * @return Policy[]
+     * @var array The content of the document
      */
-    public function getPolicies()
-    {
-        return $this->policies;
-    }
+    public $content;
 
     /**
-     * Replaces the roles associated to the profile.
-     *
-     * @param array[]|Role[] $policies List of policies descriptions or Kuzzle\Security\Role instances corresponding to the new associated roles
-     * @return Profile
+     * @var array The metadata of the document
      */
-    public function setPolicies(array $policies)
-    {
-        $this->policies = [];
-        
-        foreach ($policies as $policy) {
-            $this->policies[] = $this->policyFactory($policy);
-        }
-        
-        return $this;
-    }
+    public $meta;
 
     /**
-     * Replaces the content of the Kuzzle\Security\Profile object.
-     *
-     * @param array $content
+     * Profile constructor.
+     * @param Security $kuzzleSecurity An instantiated Kuzzle\Security object
+     * @param string $id Unique profile identifier
+     * @param array $content Profile content
+     * @param array $meta Profile metadata
      * @return Profile
      */
-    public function setContent(array $content)
+    public function __construct(Security $kuzzleSecurity, $id = '', array $content = [], array $meta = [])
```

Comment:
In https://github.com/kuzzleio/sdk-javascript/blob/6.x/src/controllers/security/profile.js, it's a Kuzzle object instead of Security. Should I do the same here ?